### DISCUSSION TITLE
AVG (personal desktop edition) reports links to free.fullspeed.tv as infected with URL:Blacklist.

### DISCUSSION THREAD

**Discussion by DiMag** [iptv-org/community#123] [Show and tell]
AVG popup message: "We have safely aborted connection on free.fullspeed.tv because it was infected with URL:Blacklist".
AVG found some m3u lists to be infected, to the extent of 2% (or close) of their links.

**Comment by freearhey** [comment]
@LaneSh4d0w @dct-infra

**Reply by BellezaEmporium** [reply]
Same for Fortinet, flagged as "malicious". dct told me it was due to some configuration inside his servers.

Nothing inside my code is malicious, it's a simple web server backed with Streamlink. Nothing more.

**Reply by dct-infra** [reply]
> Same for Fortinet, flagged as "malicious". dct told me it was due to some configuration inside his servers.
> 
> Nothing inside my code is malicious, it's a simple web server backed with Streamlink. Nothing more.

@LaneSh4d0w Nothing is malicious server side either, the issue is related to the url [URL:Blacklist".]
It seems like the url itself is the issue. I've been receiving several dmca tdns lately...

**Reply by BellezaEmporium** [reply]
> > Same for Fortinet, flagged as "malicious". dct told me it was due to some configuration inside his servers.
> > 
> > Nothing inside my code is malicious, it's a simple web server backed with Streamlink. Nothing more.
> 
> @LaneSh4d0w Nothing is malicious server side either, the issue is related to the url [URL:Blacklist".]
> It seems like the url itself is the issue. I've been receiving several dmca tdns lately...
> 
> 
> 
> 

DMCA Takedowns... that's odd... Having streams available everywhere and simply using Streamlink for scraping makes them wanna takedown everything.

**Reply by dct-infra** [reply]
As for the dmca, the fun part is that I made a template to make a decent reply explaining, they don't even bother reading. Everything seems to be automated, so I've also automated my finger on "delete".
Back to the listing, I'm quite sure the URL is submitted to Fortinet, AVG or any list they source from by the same people who see this tool as a threat to their wallet.
@DiMag  A simple solution would be to whitelist the URL, discard the error etc... there is no such thing as a malicious m3u. 
We are good honest people, you can trust us :)

**Comment by MapGuy11** [comment]
This is why only using Windows Defender is a good thing. The links are not malicious they are safe to use. But if you don't believe us that's ok you just won't have access to those channels.
